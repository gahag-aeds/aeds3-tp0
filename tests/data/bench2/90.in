90 45
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 17 28 9 9 63 44 15 67 67 33 90 37 54 66 2 63 67 77 91 93 60 86 90 8 3 92 27 58 98 39 
Add 37 66
Max 8 37
Sub 48 77
Max 59 88
Add 37 66
Sum 3 32
Add 15 44
Max 5 34
Sub 10 39
Max 23 52
Sub 50 79
Sum 12 41
Sub 12 41
Min 4 33
Sub 18 47
Min 34 63
Sub 8 37
Max 26 55
Add 3 32
Max 11 40
Add 51 80
Min 4 33
Add 57 86
Max 2 31
Sub 12 41
Max 20 49
Sub 26 55
Min 31 60
Sub 58 87
Sum 57 86
Sub 10 39
Sum 27 56
Sub 4 33
Min 49 78
Add 17 46
Sum 33 62
Add 43 72
Min 31 60
Add 29 58
Min 52 81
Add 53 82
Sum 18 47
Add 55 84
Max 5 34
Add 13 42
