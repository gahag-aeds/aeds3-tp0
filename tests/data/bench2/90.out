94
98
1508
95
94
1472
9
2
95
92
9
92
91
1
1499
1250
1
1187
1
1
1360
90
